common/xrdp_client_info.h

```diff
@@ -143,7 +143,7 @@ struct xrdp_client_info
   int use_frame_acks;
   int max_unacknowledged_frame_count;
 
-  int disableSSLv3; /* 0 = no, 1 = yes */
+  long ssl_protocols;
```

Comment:
OK, I see that we can't change size/length of existing variables. I also think tls_ciphers length is not enough because it will often be longer than 63 characters. For example: https://mozilla.github.io/server-side-tls/ssl-config-generator/

I need to rethink approach to expand tls_ciphers length.